src/test/java/org/jabref/logic/importer/fetcher/ArXivTest.java

```diff
@@ -189,7 +189,7 @@ public void searchWithMalformedIdThrowsException() throws Exception {
     public void searchIdentifierForSlicePaper() throws Exception {
         sliceTheoremPaper.clearField(FieldName.EPRINT);
 
-        assertEquals(ArXivIdentifier.parse("1405.2249v1"), finder.findIdentifier(sliceTheoremPaper));
+        assertEquals(ArXivIdentifier.parse("http://arxiv.org/abs/1405.2249v1"), finder.findIdentifier(sliceTheoremPaper));
```

Comment:
should work just with the id...

Reply:
Same reason as above, the bibtex code returns the url + id and will not be equal if not.

Reply:
Well, then the implementation needs be changed to strip the url part ;-)

Reply:
The strip was already implemented, but it checked for https and not http.